### DISCUSSION TITLE
Microsoft Designer 使用經驗分享

### DISCUSSION THREAD

**Discussion by YifenLin** [1101-GS4719-C/Python-Community-GS4719-C#1917] [General]
在生成式人工智慧（AIGC）的興起下，我們擁有了更多工具來製作精美的簡報、圖片等，以表達我們的想法，或是在社群媒體上吸引更多關注。其中一個值得介紹的網站是"Microsoft Designer"，它是一個用網頁瀏覽器開啟的平面設計工具，它導入了 ChatGPT，使我們可以使用自然語言處理的 AI 模型輕鬆製作出各式各樣的平面設計。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/138841470/7111361f-532a-4ee4-9d69-52cce14bc94d)

Microsoft Designer 提供的功能包含：圖片生成、設計製作、加入圖片、生成擦除、模糊背景、移除背景、社群媒體設計等等。該網站對這些功能做了更詳細的介紹和整理，有興趣的人可以點擊以下連結進一步了解：[Microsoft Designer介紹](https://vocus.cc/article/656c07bbfd89780001918254#data-h3-1-656c07bbfd89780001918254)。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/138841470/4c795345-bb67-4300-88c1-4250d3b688fa)

我個人使用過的功能有：圖片生成、設計製作、貼圖製作。但作為一個學生，我並沒有經常使用它，因為用文字描述圖片需求雖然更加客製化，但是AI生成一張圖片需要等待至少30秒，而且若有細節不符合要求，就需要修改文字描述並再次等待30秒，效率相對較低。因此，我建議如果你的圖片是整張簡報的重點，可以考慮使用 Microsoft Designer；但如果只是需要一些投影片的配角圖片，網路資源已經足夠使用，而且更快速（但記得註明來源）。

**Comment by benbebobo149** [comment]
我很贊同Canva Pro的選擇，它提供了許多我在創作中所需要的功能，而且使用起來非常直覺和方便。我想你也有使用過他們的產品，對於它們的易用性應該有所體會。Canva Pro不僅僅提供了設計工具，還有許多模板和素材可以使用，這使得我的創作過程更加高效和有趣。對於那些想要將設計提升到更專業水準的人來說，Canva Pro絕對是一個不錯的選擇。最後，感謝你分享了這個寶貴的建議！

**Comment by KH1126** [comment]
之前研究過一個AI人工智慧繪圖disco diffusion，
通過輸入關鍵字，可以快速得到高質量的AI影像。
相信在更熟悉如何使用AI幫忙創作後，影像工作者可以大幅提升創作效率與品質。

**Comment by William08290829** [comment]
感謝分享  看來以後簡報找圖或許不用再一直翻圖庫了
之前看過有些AI繪圖都要收費，目前這款依然是免費
但不知道哪天微軟會不會發瘋

**Comment by willychiuchiu** [comment]
哇哇哇感謝這位同學的分享，實在是太實用了！以前做簡報都亂做一通一點美感都沒有，想改進又不知道怎麼改進，現在知道了Microsoft Designer下次試試看應該可以做出一個很屌的簡報～

**Comment by wanderer657** [comment]
超感謝同學的分享，我認為這功能搭配ppt 推薦的版面設計功能，可以讓簡報呈現的精美度大幅提升，有助於演講者表達概念、情緒等等，抓住觀者的目光與提高吸引力，在提高創作自由度上很有效，可使版面的佈置更有彈性。

**Comment by yyujimmy** [comment]
根據我使用後的個人經驗，我覺得儘管 Microsoft Designer 提供了使用自然語言處理 AI 模 型的功能來製作平面設計，但它可能並不適合所有人或所有情況。對於一些專業設計師或者對於圖片的細節和品質有較高要求多的控制和自定義的功能。

此外，使用 Microsoft Designer 生成圖片所需的時間較長，在修改細節時需要重新等待的情況可能會影響工作效率。尤其是對於那些需要快速產出設計作品的人來說，這樣的等待時間可能會成為一個不小的挑戰。

因此，儘管 Microsoft Designer 提供了新穎的功能和使用方式，但對於一些專業用戶來說，仍然可能更偏向於傳統的設計工具，因為它們提供了更高的自定義性和效率。
以上我的看法!

**Comment by Eileen050824** [comment]
對我曾經使用過這個網站的人來說，我不得不承認其ppt成品是蠻有美感跟精緻的，結合AI也有為其增添使用特色。但我非常認同文中所述，AI生成文字的效率是真的有點低，主要是文字描述的精確度並沒有很好抓，起初在設定條件的時候來來回回好幾次才達成我的理想。像樓上也有提到Canva pro，我使用過也覺得那些圖庫資源就足夠，不過工具需求引人而異啦！

**Comment by yubo0oo** [comment]
我覺得這個工具真的很強大。我個人使用過的功能有：圖片生成、設計製作和貼圖製作。作為一個學生，我並沒有經常使用它，因為用文字描述圖片需求雖然更加客製化，但是AI生成一張圖片需要等待至少30秒，如果有細節不符合要求，還需要修改文字描述並再次等待30秒，效率相對較低。

因此，我建議，如果你的圖片是整張簡報的重點，可以考慮使用 Microsoft Designer；但如果只是需要一些投影片的配角圖片，網路資源已經足夠使用，而且更快速（但記得註明來源）。這樣可以更高效地完成你的簡報製作，不知道大家對這個工具有什麼看法呢？歡迎分享你的使用經驗！
